### PR TITLE
Cleanup README

### PR DESCRIPTION
**Pull request by alexjhawk**
Cleanup the readme, fix minor grammar problems, split larger steps into smaller blocks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,81 +1,92 @@
...
-HMS Ewon Ignition Integration Demo is a small suite of applications that showcase the features of integration between HMS Networks' Ewon platform, and the Inductive Automation Ignition platform.
+HMS Ewon Ignition Demo is a project to create and maintain a demo that showcases the features of integrating the HMS Networks' Ewon platform and the Inductive Automation Ignition platform.
 
-HMS Ewon Ignition Integration Demo was developed by HMS Networks using the Ewon Java Library. For more information on this demo and developing applications with the Ewon Java Library, please contact us at [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se") and/or visit the Ewon Developer website at [https://developer.ewon.biz/](https://developer.ewon.biz/ "https://developer.ewon.biz/")
+This project was developed and is maintained by HMS Networks, using the Ewon Java library. For more information about this demo and/or developing applications with the Ewon Java library, please contact us at [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se"), visit the Ewon Developer website at [https://developer.ewon.biz/](https://developer.ewon.biz/ "https://developer.ewon.biz/")
```
Comment (oliver-walker-hms) [inline README.md]: I think the grammar looks good. You could probably add some sort of table of contents here to jump to headers

Comment (alexjhawk) [reply README.md]: added

File: README.md [hunk 2]
```diff
@@ -1,83 +1,104 @@
...
-If no information is available on the Ewon Ignition Integration Demo Wiki, or if your issue persists, please create a GitHub issue, found here: [https://github.com/hms-networks/FlexyIgnitionDemo/issues](https://github.com/hms-networks/FlexyIgnitionDemo/issues "hms-networks/FlexyIgnitionDemo/issues")
-
-## Getting Started ##
-To complete installation, please perform all the setup steps in the sections below! To begin, the following items are required:
-
-- HMS Ewon Product
-- Computer with Administrator Privileges
+HMS Ewon Ignition Demo is a project to create and maintain a demo that showcases the features of integrating the HMS Networks' Ewon platform and the Inductive Automation Ignition platform.
+
+This project was developed and is maintained by HMS Networks, using the Ewon Java library. For more information about this demo and/or developing applications with the Ewon Java library, please contact us at [us-services@hms.se](mailto:us-services@hms.se "us-services@hms.se"), visit the Ewon Developer website at [https://developer.ewon.biz/](https://developer.ewon.biz/ "https://developer.ewon.biz/")
```
Comment (TomKimsey) [inline README.md]: > , please contact us at ... , visit the Ewon Developer website at ....

This is a little awkward without the or

File: README.md [hunk 3]
```diff
@@ -1,83 +1,104 @@
...
+    - *Flexy Base Unit*: [https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28](https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28 "https://websupport.ewon.biz/sites/default/files/ig-014-0-en-ewon_flexy_-_base_units.pdf#page=28")
+- Computer with Software Installation Privileges
 - Talk2M API Key
   - If you do not have a Talk2M/Ewon API Key, you may request one here: [https://developer.ewon.biz/registration](https://developer.ewon.biz/registration "https://developer.ewon.biz/registration").
-- Demo Resource Pack (.zip)
-	- Download Here: [https://github.com/hms-networks/FlexyIgnitionDemo/releases](https://github.com/hms-networks/FlexyIgnitionDemo/releases "https://github.com/hms-networks/FlexyIgnitionDemo/releases")
+- Ewon Ignition Demo Resource Pack (.zip)
+	- Download Here: [https://github.com/hms-networks/FlexyIgnitionDemo/releases](https://github.com/hms-networks/FlexyIgnitionDemo/releases "https://github.com/hms-networks/FlexyIgnitionDemo/releases"
```
Comment (TomKimsey) [inline README.md]: Did you mean to add this tab here?

File: README.md (hunk 2 above)
Comment (alexjhawk) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (alexjhawk) [reply README.md]: Before "- Download Here"? If so, fixed

**Pull request merged**